lib/arke/strategy/base.rb

```diff
@@ -0,0 +1,71 @@
+require 'exchange'
+
+module Arke::Strategy
+  class Base
+    RunNotOverriden = Class.new(StandardError)
+
+    def initialize
```

Comment:
Initialize takes the config, it allow easier test we no dependency to configuration you can create strategy with hash params, pass the whole config hash to new
Initialize will set all needed class variables.

Start will start the needed threads.